src/plugins/vis_type_table/public/utils/use_pagination.ts

```diff
@@ -6,10 +6,10 @@
 import { useCallback, useEffect, useMemo, useState } from 'react';
 import { TableVisConfig } from '../types';
 
-export const usePagination = (visConfig: TableVisConfig, nrow: number) => {
+export const usePagination = (visConfig: TableVisConfig, nRow: number) => {
   const [pagination, setPagination] = useState({
     pageIndex: 0,
-    pageSize: visConfig.perPage,
+    pageSize: visConfig.perPage || 0,
```

Comment:
can we just define `perPage` up here so we just have the logical OR fallback once?

Reply:
yep, due to `perPage` type is defined ad `number | ''`